db/seeds.rb

```diff
@@ -4,4 +4,19 @@
 # Examples:
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
\ No newline at end of file
+#   Character.create(name: "Luke", movie: movies.first)
+
+# Users
+first_user = User.create(name: 'Tom', photo: 'https://images.unsplash.com/photo-1607990281513-2c110a25bd8c?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=634&q=80', bio: 'Teacher from Mexico.', posts_counter: 0)
+second_user = User.create(name: 'Lilly', photo: 'https://images.unsplash.com/photo-1506863530036-1efeddceb993?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=744&q=80', bio: 'Teacher from Poland.', posts_counter: 0)
+third_user = User.create(name: 'Tim', photo: 'https://images.unsplash.com/photo-1522075469751-3a6694fb2f61?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=580&q=80', bio: 'Teacher from Canada.', posts_counter: 0)
+
+# Posts
+first_post = Post.create(author: first_user, title: 'Hello', text: 'This is my first post', comments_counter: 0, likes_counter: 0)
+second_post = Post.create(author: first_user, title: 'Hello again', text: 'This is my second post', comments_counter: 0, likes_counter: 0)
+third_post = Post.create(author: first_user, title: 'Hello again and again', text: 'This is my third post', comments_counter: 0, likes_counter: 0)
+fourth_post = Post.create(author: first_user, title: 'Hello', text: 'This is my fourth post', comments_counter: 0, likes_counter: 0)
+
+fifth_post = Post.create(author: second_user, title: 'Hello', text: 'This is my first post', comments_counter: 0, likes_counter: 0)
+sixth_post = Post.create(author: second_user, title: 'Hello again', text: 'This is my second post', comments_counter: 0, likes_counter: 0)
+seventh_post = Post.create(author: second_user, title: 'Hello again and again', text: 'This is my third post', comments_counter: 0, likes_counter: 0)
```

Comment:
- [ ] There are some issues happening when seeding these data. Let's fix that
<img width="759" alt="Screen Shot 2022-10-01 at 9 59 49 AM" src="https://user-images.githubusercontent.com/40503704/193401932-cacaba2e-a031-4120-8767-93b485ec6718.png">